repository_service_tuf_api/bootstrap.py

```diff
@@ -53,6 +54,7 @@ class Config:
 
 class PostData(BaseModel):
     task_id: Optional[str]
+    last_update: datetime
```

Comment:
I don't mention the `status`, but the `state`, isn't?
If yes,  I mentioned because we know that all task submitted we know the initial status.
`PENDING`. See https://docs.celeryq.dev/en/stable/reference/celery.states.html#celery.states.PENDING